translations/de/info/task_description.html

```diff
@@ -1,68 +1,105 @@
 <img class="for_info_only half" style="float: left" src="{{ MEDIA }}cipher_map_illustration.png">
 
 <div class="story">
-    <p>"Nikola, A1213pokl, bAse730onE."</p>
-
-    <p>"Yes Sofia, what is it?"</p>
-
-    <p>"You tell me! Your torture device is singing my circuits with its new lexicon. There is no way I can remember
-        these new passwords and the thing doesn't accept simple and easy ones!"</p>
-
-    <p>"Oh, those will be good passwords, you can use them."</p>
-
-    <p>"Why can’t you use them!?" Sofia asked almost hysterically. "I’ve already forgotten them! Do you want me locked
-        out of my own house for eternity? Come up with something easier so I don’t have to keep all that randomized
-        gobbledygook in my head." Sofia rarely acted so demanding but she had reached her boiling point. The stress of
-        the past few days coupled with her exhaustion had brought her to the edge.</p>
-
-    <p>"Don’t worry, I was expecting something easier to remember. Why don’t you use the cipher map to help with your
-        password.
-        With it we can encrypt all the passwords and leave them right next to the door or in a special place that we
-        all agree upon. You will only be able to read them with the use of the cipher map which we will take with us
-        on our trip.</p>
-
-    <p>"So I don’t need to remember all of those passwords? All we have to do is make sure that we don't lose the
-        cipher map." asked Sofia with hopefully.</p>
-
-    <p>"Yes, that is correct."</p>
-
-    <p>"Awesome. Show me the cipher map and explain how this works one more time."</p>
+    <p>
+        »Nikola, A1213pokl, bAse730onE.«
+    </p>
+
+    <p>
+        »Ja Sofia, was ist los?«
+    </p>
+
+    <p>
+        »Sag Du es mir! Dein Foltergerät, mit seinem neuen Lexikon, lässt meine
+        Schaltkreise qualmen. Ich kann mir diese komplizierten neuen Passwörter einfach
+        nicht merken und das Ding will und will keine einfacheren akzeptieren.«
+    </p>
+
+    <p>
+        »Oh, aber das sind gute Passwörter, und natürlich kannst Du sie verwenden.«
+    </p>
+
+    <p>
+        »Warum kannst Du sie nicht benutzen!?«, fragte Sofia beinahe hysterisch.<br>
+        »Ich habe sie schon vergessen! Willst Du, dass ich für alle Ewigkeit aus meinem
+        eigenen Haus ausgeschlossen bin? Lass Dir gefälligst was Einfacheres einfallen,
+        sodass ich mir nicht diesen zufälligen Kauderwelsch merken muss.«, keiffte Sophia.<br>
```

Comment:
Sofia (instead Sophia), but I recommend to end the speak without mentioning her again anyway. It's done before, it is clear Sofia is still speaking. Your choice of a verb isn't helping too :)